listenbrainz/webserver/static/js/src/types.d.ts

```diff
@@ -126,6 +126,11 @@ declare type Alert = {
   message: string | JSX.Element;
 };
 
+// Expect either a string or an Error or an html Response object
+declare type ErrorForAlert =
```

Comment:
ErrorForAlert confuses me (IDK why?). I cant think of a better name too. Sorry.

Reply:
I agree, not the best naming. I'll think of something.